bin/rbenv-each

```diff
@@ -13,21 +13,25 @@ set -e
 [ -n "$RBENV_DEBUG" ] && set -x
 
 # Provide rbenv completions
-if [ "$1" = "--complete" ]; then
-  echo -v
-  exit
-fi
-
-if [ "$1" = "-v" ]; then
-  verbose=1
-  shift
-fi
-
 case "$1" in
-""  | -* )
-  rbenv-help --usage each >&2
-  exit 1
-  ;;
+  --complete )
+    echo --help
+    echo --verbose
+    rbenv shims --short
```

Comment:
I don't see a value in autocompleting shims. Most of the time people will just run `ruby` or `rake` which is not hard to type. I feel that we should not provide completions for `rbenv each` arguments other than its flags.